js/samples/quickstart-kotlin/app/src/main/res/layout/activity_main.xml

```diff
@@ -39,7 +39,7 @@
         <ScrollView
             android:id="@+id/ContentPane"
             android:layout_width="match_parent"
-            android:layout_height="480dp"
+            android:layout_height="251dp"
```

Comment:
change seems unusual...

Reply:
I reduced the panel size to adjust the screen because added additional buttons.